### ISSUE TITLE
references to 3.1 in a11y techniques

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#1285]
The media overlays reference should be "Media Overlays 3" not "Media Overlays 3.1". The link correctly points to the "latest" URL.

The EPUB overview reference should drop the .1 and use a "latest" URL.

**Comment by mattgarrish** [comment]
These are now fixed in the respec-converted version of the document: https://w3c.github.io/publ-epub-revision/a11y/techniques/techniques.html
